lib/ecl/tests/ecl_smspec.c

```diff
@@ -23,6 +23,17 @@
 
 #include <ert/ecl/ecl_smspec.h>
 
+void test_sort( ecl_smspec_type * smspec )
+{
+  ecl_smspec_sort( smspec );
```

Comment:
This is my paranoia speaking, but could you add
```c
test_assert_true(ecl_smspec_num_nodes(smspec) == 10);
```
for `10` being the correct size of `smspec`?